src/server/Settings.cpp

```diff
@@ -0,0 +1,51 @@
+#include <server/Settings.hpp>
+#include <fstream>
+#include <iostream>
+
+void Settings::LoadFromFile(const std::filesystem::path& filePath) {
+    std::ifstream file(filePath);
+
+    if (!file.is_open()) {
+        std::cerr << "Error: Could not open configuration file." << std::endl;
+        return;
+    }
+
+    // Parse the JSON file
+    nlohmann::json jsonConfig;
+    file >> jsonConfig;
+
+    // Extract lobby settings
+    if (jsonConfig.contains("lobby")) {
+        const auto& lobby = jsonConfig["lobby"];
+        _lobbySettings.address = lobby.value("address", "127.0.0.1"); // Default to localhost
```

Comment:
Actually make it localhost, not loopback. Same for the ranch port. 
```suggestion
        // Default to localhost
        _lobbySettings.address = lobby.value("address", "localhost"); 
```

Reply:
Thank you.